### PR TITLE
GPS Updated..

### PR DESCRIPTION
**Pull request by Santhosh-M31**

### CODE REVIEW

File: spatial-data-viz.Rmd [hunk 1]
```diff
@@ -1,1328 +1,1372 @@
...
+```{r echo=FALSE, fig.align='center', out.width='60%'}
+knitr::include_graphics('images/spatial_data_viz/gps23.png')
+```
+
+24. Once you have the individual frames, you can use a program such as [ezgif.com](https://ezgif.com/) to create an animated GIF from them.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps24.png')
+```
+
```
Comment (spatialthoughts) [inline spatial-data-viz.Rmd:591]: Add the resulting GIF (file is in the aciwrm folder) and link it as it is done previously

Comment (Santhosh-M31) [reply spatial-data-viz.Rmd:585]: GIF linked.

File: spatial-data-viz.Rmd [hunk 2]
```diff
@@ -1,1328 +1,1372 @@
...
+### Exercise: Visualize GPS Tracks
+
+GPS tracks have become ubiquitous in modern life. With GPS built-into most phones, many of us capture the tracks while running or biking outdoors. Cab companies use GPS tracks collected during the trip to determine fares. Delivery and logistics companies store and analyze millions of GPS tracks from their assets to derive location intelligence. 
+
+We will use a GPS track I collected using the open-source [GPS Logger app](https://www.basicairdata.eu/projects/android/android-gps-logger/) while cycling to work. The default format for storing GPS tracks is [GPS Exchange Format (GPX)](https://en.wikipedia.org/wiki/GPS_Exchange_Format). It is a XML-based text format that allows storing points, tracks and routes in a single file. We will use the data in ``sample_gps_track.gpx`` file and create an animated GIF showing the trip.
+
+1. Locate the ``sample_gps_track.gpx`` file in the `animating_timeseries/` folder and drag it to the canvas.
```
[the remark this answers is not in the document]
Comment (Santhosh-M31) [reply spatial-data-viz.Rmd]: Fixed the Path .

**Comment by Santhosh-M31** [comment]
Have updated the mentioned comments. 
PL 7be2303 is latest.

**Comment by Santhosh-M31** [comment]
Have updated the mentioned comments.
PL e466d42  is latest.

**Pull request merged**